txeventq/kafka-connector-example/sql/setupTeq.sql

```diff
@@ -0,0 +1,20 @@
+set serveroutput on
+
+alter pluggable database all close immediate;
+drop pluggable database cdb1_pdb1 including datafiles;
```

Comment:
I think we should also prompt for the Pluggable database name.
If a user just blindly runs this SQL file and has an existing `cdb1_pdb1` pluggable database, this script will just drop it.

Reply:
@gvenzl I have made the suggested changes.